src/main/java/org/jitsi/videobridge/VideoChannel.java

```diff
@@ -629,23 +442,19 @@ void sctpConnectionReady(Endpoint endpoint)
 
         if (endpoint.equals(getEndpoint()))
         {
-            if (lastNController.getLastN() >= 0 ||
-                    lastNController.getCurrentLastN() >= 0)
+            if (lastN > 0)
```

Comment:
Thanks @bgrozev. No, it doesn't and I think I've completely broken the lastN notifications..